server/log_operation_manager_test.go

```diff
@@ -254,13 +255,17 @@ func TestMasterFor(t *testing.T) {
 
 type masterForEvenFactory struct{}
 
-func (m masterForEvenFactory) NewElection(ctx context.Context, treeID int64) (election.MasterElection, error) {
-	isMaster := (treeID % 2) == 0
+func (m masterForEvenFactory) NewElection(ctx context.Context, treeID string) (election.MasterElection, error) {
```

Comment:
Does KT's `domainID` identify a tree, more specifically is `itoa(treeID) == domainID`? Even if so, could we rename `treeID` here to something more general, like `resourceID` (and point out somewhere in the comments that `treeID` in Trillian is an example of a master-elected resource)? I think it doesn't matter much on this level of abstraction whether the resource is a tree or something else.

Reply:
The answer to your direct question is a qualified yes: a KT `domainID` has a 1:2 relationship with TreeID.  (1 domainID has 1 Trillian Log and 1 Trillian Map).

More generally, I'm hoping to use `util/election` inside KT for the sequencing and batching of mutations.  `resourceID` might indeed be a better name.

Reply:
Please update `treeID` to `resourceID` in this file as well.

Reply:
The IDs in this context are `int64`s, so I kept them as logIDs to be consistent.